app/Http/Controllers/LoadoutsController.php

```diff
@@ -45,6 +46,21 @@ public function preview($id)
         return view('loadouts.preview')->withLoadout($loadout);
     }
 
+    public function delete($id)
```

Comment:
We can use a native Request here to validate and ensure user has permissions.

Reply:
Can you give an example? Not sure what you mean

Reply:
https://laravel.com/docs/8.x/validation this should have some good examples for ya.

Reply:
Let me know if I implemented what you were thinking of via this commit: https://github.com/drg-tools/karl.gg/pull/202/commits/6b671d944238c54129498b46b140e1a0ed4d6d5b

Thanks